test/NuGet.Core.Tests/NuGet.ProjectModel.Test/PackageSpecCloningTests.cs

```diff
@@ -638,14 +641,53 @@ public void TargetFrameworkInformationCloneTest()
 
             // Assert
             Assert.Equal(originalTargetFrameworkInformation, cloneToTestDependencies);
-            Assert.False(object.ReferenceEquals(originalTargetFrameworkInformation, cloneToTestDependencies));
+            Assert.False(ReferenceEquals(originalTargetFrameworkInformation, cloneToTestDependencies));
 
             // Act
             originalTargetFrameworkInformation.Dependencies.Clear();
 
             // Assert
             Assert.NotEqual(originalTargetFrameworkInformation, cloneToTestDependencies);
             Assert.Equal(1, cloneToTestDependencies.Dependencies.Count);
+
+            //Act
+            var cloneToTestDownloadDependencies = originalTargetFrameworkInformation.Clone();
+
+            // Assert
+            Assert.Equal(originalTargetFrameworkInformation, cloneToTestDownloadDependencies);
```

Comment:
Each test should be a separate method with a good name that describes the scenario and expected result (e.g.:  `NameOfMemberUnderTest_WhenArgumentIsNull_Throws`).